app/Helpers/database/leaderboard.php

```diff
@@ -309,14 +310,14 @@ function UploadNewLeaderboard(
     $displayOrder = 0;
     $originalAuthor = null;
 
+    /** @var ?Leaderboard $foundLeaderboard */
+    $foundLeaderboard = null;
+
     if ($idInOut > 0) {
         $foundLeaderboard = Leaderboard::find($idInOut);
         if ($foundLeaderboard) {
             $displayOrder = $foundLeaderboard->order_column;
-            $originalAuthor = $foundLeaderboard->authorUser;
-
-            $data['DisplayOrder'] = $displayOrder;
-            $data['Author'] = $originalAuthor?->display_name ?? "Unknown";
+            $originalAuthor = $foundLeaderboard->developer;
```

Comment:
This is absorbed from https://github.com/RetroAchievements/RAWeb/pull/2993. I discovered that issue while doing manual testing for all this stuff.